### DISCUSSION TITLE
Channel points rewards

### DISCUSSION THREAD

**Discussion by Zelassin** [baffler/Transparent-Twitch-Chat-Overlay#72] [Ideas]
This chat overlay is great and definitely the best out of all the available options for seeing your chat while streaming on a single monitor setup. Only problem i have with this is that i cannot see what my viewers redeem with their points unless i use the twitch popout chat option, which looks very ugly. Is it possible to combine KapChat's looks with points rewards?

**Comment by baffler** [comment]
I'll look into maybe getting point redemptions to show up in chat for KapChat, but not sure about that. But I can make the twitch popout chat look better. What did you have in mind? Is it mainly the font? Cause I can make it be more like KapChat with bold text, slightly bigger font and some shadow on the text.

**Reply by Zelassin** [reply]
Not sure if the custom CSS thing is working correctly for me, because it basically looks like the regular twitch popout. I can still see the "send a message" and "Stream chat" fields on top and bottom that i'd very much like to be cropped out completely, only leaving the actual messages field like in KapChat, and the background is not completely transparent. I managed to remove the gray tint by changing the "0, 0, 0, 0.1" value in body background to "0, 0, 0, 0", but the rest is still a problem for me. Another thing that i think should work differently - the custom CSS field does not save what values I entered in it, it displays the default text every time i open settings (even though the changes i've made do apply to the chat window - it just doesn't save the text). 
And then there's obviously the FFZ/BTTV emote support and the difference in font types and size. 
Here's a comparison between the two types of chat on my end:
![image](https://user-images.githubusercontent.com/61279769/106828427-3fb93b00-669b-11eb-84e5-f6acd6985b26.png)
![image](https://user-images.githubusercontent.com/61279769/106828596-945cb600-669b-11eb-93f5-41ba6f49843c.png)

Now i obviously don't expect you to implement any of that for a random request, i just listed what are in my opinion the biggest problems with the popout chat for me.
